frontend/src/app/okrview/keyresult/key-result-form/key-result-milestone-form/key-result-milestone-form.component.html

```diff
@@ -1,21 +1,15 @@
 <mat-slide-toggle class="toggleMilestone" [checked]="isChecked" [disabled]="formArray?.length > 0" (change)="isChecked=$event.checked">{{ 'key-result-milestone-form.toggle' | translate }}</mat-slide-toggle>
 
-<button class="addMilestone-button" mat-mini-fab matTooltip="{{ 'key-result-milestone-form.tooltip.add-button' | translate }}" (click)="addMilestone(undefined, undefined,undefined, undefined)" *ngIf="isChecked">
-  <mat-icon>
-    add_circle_outline
-  </mat-icon>
-</button>
-
 <div *ngIf="isChecked" class="keyresult-milestone-form-wrapper">
   <div class="inputs" *ngFor="let formGroup of formArray.controls">
     <mat-form-field class="text-input">
-      <input type="text" matInput placeholder="{{ 'key-result-milestone-form.name-*' | translate }}" [formControl]="formGroup.get('name')">
+      <input type="text" matInput placeholder="{{ 'key-result-milestone-form.name-*' | translate }}" [formControl]="$any(formGroup.get('name'))">
```

Comment:
Wieso taucht hier ein $any() auf

Reply:
Wollte diese Fehlermeldung fixen: "Type AbstractControl is not assignable to type FormControl" und es einheitlich wie unten machen wo: "[formControl]="$any(formGroup.get('value'))" steht, aber ist jetzt wieder rückgängig gemacht.